zeebe/backup/src/main/java/io/camunda/zeebe/backup/management/BackupService.java

```diff
@@ -44,13 +45,14 @@ public BackupService(
       final BackupStore backupStore,
       final PersistedSnapshotStore snapshotStore,
       final Path segmentsDirectory,
-      final JournalInfoProvider raftMetadataProvider) {
+      final JournalInfoProvider raftMetadataProvider,
+      final MeterRegistry brokerMeterRegistry) {
     this.nodeId = nodeId;
     this.partitionId = partitionId;
     this.numberOfPartitions = numberOfPartitions;
     this.snapshotStore = snapshotStore;
     this.segmentsDirectory = segmentsDirectory;
-    metrics = new BackupManagerMetrics(partitionId);
+    metrics = new BackupManagerMetrics(partitionId, brokerMeterRegistry);
```

Comment:
:thought_balloon: Given that we have a partition id here, I'm 99% sure that we should use the partition scoped registry here, not the broker one.

Reply:
I was a bit hesitant to avoid changing the code too much, but it's just probably going to be simpler :+1: